### PR TITLE
Release/8.10.4 - Policy numbering

### PR DESCRIPTION
**Pull request by AmbyrElan**
Releasing the policy numbering style changes to base.

### CODE REVIEW

**Comment by nickdenardis** [comment]
I'm good with this release after there is a write-up of the changes. 

Can you outline the items part of this release in two categories?

1. End-user/template changes/additions
2. Structural changes for developers to know

**Review by nickdenardis — APPROVED** [review]

**Review by breakdancingcat — APPROVED** [review]

File: styleguide/Views/styleguide-cms-list-styles.blade.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <h2>Policy lists</h2>
+
+        <p>Policy numbering list style is a nested ordered and unordered list style.</br>
+        Apply the <code>policy-numbering</code> class to the opening ordered list element.</br>
+        Each new nested list should start within the previous list item.</p>
+
+        <code class="w-1/2">
+            <ul style="list-style: none">
+                <li>&lt;ol class="policy-numbering"&gt;</li>
+                <li>&#20;&#20;&#20;&#20;&lt;li&gt;Item 1.0&lt;/li&gt;</li>
```
Comment (breakdancingcat) [inline styleguide/Views/styleguide-cms-list-styles.blade.php:54]: Shouldn't all these be added by css?

File: resources/scss/components/_flag.scss [hunk 2]
```diff
@@ -1,41 +1,73 @@
...
+                clip-path: polygon(0 0, 100% 0, 100% 4%, 50% 14%, 0 4%);
+                //clip-path: polygon(0 0, 100% 0, 100% 12%, 50% 4%, 0 12%);
+                transition-property: top;
+                width: 100%;
+                height: 8rem;
+                top: 24px;
+                content: "";
+                left: 0;
+
+                /*
```
Comment (nickdenardis) [inline resources/scss/components/_flag.scss:50]: Let's remove this commented-out style.

File: resources/views/components/accordion.blade.php [hunk 3]
```diff
@@ -5,7 +5,18 @@
...
         <li>
             <a href="#definition-{{ $item['promo_item_id'] }}" id="definition-{{ $item['promo_item_id'] }}"><span aria-hidden="true"></span>{{ $item['title'] }}</a>
-            <div class="content">{!! $item['description'] !!}</div>
+            <div class="content">
+                <div class="flex gap-6 items-center flex-col {{ !empty($item['option']) && ($item['option'] === 'Left' || $item['option'] === 'Center') ? (($item['option'] === 'Center') ? '' : 'mt:flex-row-reverse mt:items-start') : 'mt:items-start mt:flex-row'}}">
+                    <div class="w-full">{!! $item['description'] !!}</div>
+
+                    @if(!empty($item['relative_url']))
+                        <figure class="m-0 w-2/3 shrink-0 {{ !empty($item['option']) && $item['option'] === 'Center' ? '' : 'mt:w-2/5' }}">
+                            <img src="{{ $item['relative_url'] }}" alt="{{ $item['filename_alt_text'] }}">
```
Comment (nickdenardis) [inline resources/views/components/accordion.blade.php:14]: What is the guidelines we are following for lazy loading images? Anything beyond the hero?

File: resources/views/homepage.blade.php [hunk 4]
```diff
@@ -7,6 +7,11 @@
         {!! $base['page']['content']['main'] !!}
     </div>
 
+    @if(!empty($components['catalog-1']['data']))
+        <div>
+            @include('components.catalog', ['data' => $components['catalog-1']['data'], 'component' => $components['catalog-1']['component'], 'class' => 'my-class'])
```
Comment (nickdenardis) [inline resources/views/homepage.blade.php:12]: What is `my-class` here?

**Review by nickdenardis — CHANGES_REQUESTED** [review]
A few inline comments in addition to the need for a write up of the changes in this release since it appears to be more than only policy numbering addition.

**Comment by breakdancingcat** [comment]
I believe these are incomplete pieces that I left behind, I will close this out and create a new release with the latest changes

**Pull request closed without merging**